lib/active_remote/association.rb

```diff
@@ -131,10 +131,10 @@ def validate_scoped_attributes(associated_class, object_class, options)
         raise "Could not find attribute: '#{options[:scope]}' on #{associated_class}" unless associated_class.public_instance_methods.include?(options[:scope])
       end
 
-    private
```

Comment:
@mmmries would like a word with you. 😂

Reply:
👋 hi friends

Reply:
I've left the mad rubocop behind. I'm all about that zen rubocop now

Reply:
Hahahaha